### PR TITLE
Moved every block to a module from useTasks

### PR DESCRIPTION
**Pull request by Omri-Levy**
### Description
Each block is now wrapped in a React hook.

### CODE REVIEW

File: apps/backoffice-v2/src/pages/Entity/components/BadgeCell/BadgeCell.tsx [hunk 1]
```diff
@@ -18,3 +18,4 @@ export const BadgeCell = React.forwardRef<
     </AnimatePresence>
   );
 });
+BadgeCell.displayName = 'BadgeCell';
```
Comment (Omri-Levy) [inline apps/backoffice-v2/src/pages/Entity/components/BadgeCell/BadgeCell.tsx:21]: Fixes lint issue.

File: apps/backoffice-v2/src/pages/Entity/components/BadgeCell/BadgeCell.tsx [hunk 2]
```diff
@@ -1,10 +1,10 @@
 import { AnimatePresence } from 'framer-motion';
-import { ComponentProps } from 'react';
+import { ComponentProps, ElementRef, forwardRef } from 'react';
 import { MotionBadge } from '../../../../common/components/molecules/MotionBadge/MotionBadge';
 import * as React from 'react';
 
-export const BadgeCell = React.forwardRef<
-  HTMLDivElement,
+export const BadgeCell = forwardRef<
+  ElementRef<typeof MotionBadge>,
```
Comment (Omri-Levy) [inline apps/backoffice-v2/src/pages/Entity/components/BadgeCell/BadgeCell.tsx:7]: Previous type was incorrect.

File: apps/backoffice-v2/src/pages/Entity/hooks/useTasks/hooks/useAssosciatedCompaniesBlock/useAssociatedCompaniesBlock.stories.tsx [hunk 3]
```diff
@@ -3,9 +3,9 @@ import { Case } from '@/pages/Entity/components/Case/Case';
 import { Card } from '@/common/components/atoms/Card/Card';
 import { ctw } from '@/common/utils/ctw/ctw';
 import { CardContent } from '@/common/components/atoms/Card/Card.Content';
-import { cells } from '@/pages/Entity/hooks/useEntity/cells';
 import { useAssociatedCompaniesBlock } from '@/pages/Entity/hooks/useTasks/hooks/useAssosciatedCompaniesBlock/useAssociatedCompaniesBlock';
 import { FunctionComponent } from 'react';
+import { cells } from '../../../useEntity/cells';
```
Comment (tomer-shvadron) [inline apps/backoffice-v2/src/pages/Entity/hooks/useTasks/hooks/useAssosciatedCompaniesBlock/useAssociatedCompaniesBlock.stories.tsx:8]: Now it doesn't even make sense. Previous file changed to '@' notation, here to dot notation

File: apps/backoffice-v2/src/pages/Entity/hooks/useTasks/hooks/useBankingDetailsBlock/useBankingDetailsBlock.tsx [hunk 4]
```diff
@@ -0,0 +1,33 @@
+export const useBankingDetailsBlock = ({ bankDetails, workflow }) => {
```
Comment (tomer-shvadron) [inline apps/backoffice-v2/src/pages/Entity/hooks/useTasks/hooks/useBankingDetailsBlock/useBankingDetailsBlock.tsx:1]: This is a hook that doesn't use any native hooks? Is that necessary?

File: apps/backoffice-v2/src/pages/Entity/hooks/useTasks/hooks/useCompanySanctionsBlock/useCompanySanctionsBlock.tsx [hunk 5]
```diff
@@ -0,0 +1,296 @@
+import { Badge } from '@ballerine/ui';
+import { ctw } from '@/common/utils/ctw/ctw';
+import { WarningFilledSvg } from '@/common/components/atoms/icons';
+import { toTitleCase } from 'string-ts';
+import { isValidUrl } from '@/common/utils/is-valid-url';
+import * as React from 'react';
+
+export const useCompanySanctionsBlock = companySanctions => {
```
Comment (tomer-shvadron) [inline apps/backoffice-v2/src/pages/Entity/hooks/useTasks/hooks/useCompanySanctionsBlock/useCompanySanctionsBlock.tsx:8]: Same here

File: apps/backoffice-v2/src/pages/Entity/hooks/useTasks/hooks/useDirectorsRegistryProvidedBlock/useDirectorsRegistryProvidedBlock.tsx [hunk 6]
```diff
@@ -0,0 +1,39 @@
+export const useDirectorsRegistryProvidedBlock = directorsRegistryProvided => {
```
Comment (tomer-shvadron) [inline apps/backoffice-v2/src/pages/Entity/hooks/useTasks/hooks/useDirectorsRegistryProvidedBlock/useDirectorsRegistryProvidedBlock.tsx:1]: Samesies

**Review by tomer-shvadron — APPROVED** [review]

**Pull request merged**